shell/platform/tizen/flutter_tizen_engine_unittest.cc

```diff
@@ -65,49 +34,53 @@ class FlutterTizenEngineTest : public ::testing::Test {
     engine_ = nullptr;
   }
 
-  std::string assets_path_;
-  std::string icu_data_path_;
-  std::string aot_lib_path_;
   flutter::FlutterTizenEngine* engine_;
-  FlutterDesktopEngineProperties engine_prop_ = {};
-  std::vector<const char*> switches_;
+};
+
+class FlutterTizenEngineTestHeaded : public FlutterTizenEngineTest {
+ protected:
+  void SetUp() {
+    FlutterTizenEngineTest::SetUp();
+    engine_->InitializeRenderer(0, 0, 800, 600);
+  }
 };
 
 TEST_F(FlutterTizenEngineTest, Run) {
   EXPECT_TRUE(engine_ != nullptr);
-  EXPECT_TRUE(engine_->RunEngine(engine_prop_));
+  EXPECT_TRUE(engine_->RunEngine());
   EXPECT_TRUE(true);
 }
 
 // TODO
 TEST_F(FlutterTizenEngineTest, DISABLED_Run_Twice) {
```

Comment:
Remain this TC disabled , I guess you must have some comment regarding this TC. :smile: 
Honestly,I don't know either(I think I got it from somewhere...). You can delete it if you think you don't need it.

Reply:
I thought you were going to make the change in https://github.com/flutter-tizen/engine/pull/135. I'll re-enable it.